style.css

```diff
@@ -1,1258 +1,1961 @@
+  color: #fff;
+  position: absolute;
+  width: 801px;
+  height: 56px;
+  left: -212px;
+  top: 116px;
   font-style: normal;
-  font-weight: 400;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 52px;
+}
+
+.name-container input {
+  font-family: Poppins, sans-serif;
+  border: 1px solid transparent;
+  border-radius: 8px;
+  background: #fff;
   font-size: 17px;
   line-height: 20px;
-  display: flex;
-  align-items: center;
   color: #172b4d;
+  width: 80%;
+  margin-top: 77px;
+  height: 40px;
+  padding-left: 5px;
 }
 
-.user-email ::placeholder {
-  position: absolute;
-  left: 16px;
-  right: 38px;
-  top: 10px;
+.mail-container input {
   font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 400;
+  border: 1px solid transparent;
+  border-radius: 8px;
+  background: #fff;
   font-size: 17px;
   line-height: 20px;
-  display: flex;
-  align-items: center;
   color: #172b4d;
+  width: 80%;
+  margin-top: -2px;
+  height: 40px;
+  padding-left: 5px;
 }
 
-textarea {
-  position: absolute;
-  left: 42px;
-  top: 480px;
-  bottom: 10px;
-  width: 320px;
-  height: 170px;
+.textarea-container textarea {
+  font-family: Poppins, sans-serif;
+  border: 1px solid transparent;
   border-radius: 8px;
-  margin-top: 20px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  display: flex;
-  color: #b3bac5;
-}
-
-input {
-  position: relative;
-  font-family: "Poppins", sans-serif;
-  font-weight: 100;
+  background: #fff;
   font-size: 17px;
   line-height: 20px;
-  padding: 10px;
-  width: 322px;
-  top: 800px;
-  height: 48px;
-  left: 41px;
-  border-radius: 8px;
-  margin-bottom: 20px;
-  background-color: #fff;
-  border: solid #b3bac5;
-  border-width: 1px;
+  color: #172b4d;
+  width: 80%;
+  margin-top: -2px;
+  height: 170px;
+  padding-left: 5px;
 }
 
-.submit-btn {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 12px 16px;
-  position: absolute;
-  width: 146px;
+button {
+  width: 140px;
   height: 48px;
-  left: 42px;
-  top: 690px;
-  border-radius: 8px;
-  font-family: "Poppins", sans-serif;
+  margin-left: -153px;
+  border: 1px solid #6070ff;
+  font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 17px;
   line-height: 24px;
+  background: #fff;
+  border-radius: 8px;
+  color: #6070ff;
   text-align: center;
   letter-spacing: 0.03em;
-  color: #6070ff;
-  background-color: #fff;
-  border: solid #fff;
-  border-width: 1px;
-}
-
-.contact-img3 {
-  position: absolute;
-  width: 134px;
-  height: 5px;
-  left: 129px;
-  bottom: 8px;
-  background: #0d151e;
-  border-radius: 10px;
-  top: 767px;
 }
 
-.contact button:hover {
-  color: white;
-  background: #6070ff;
-  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-}
-
-.contact button:active {
-  background: #2230d2;
-  color: white;
+@media only screen and (min-width: 768px) {
```

Comment:
The design here is fine 💯 but the transition is not smooth enough, even if below `768px` it goes to mobile version but the mobile version is supposed to be looking as good as it did before, or maybe think of adding an intermediate version between mobile and web, because your design looks good at `768px` but there's room for improvement at `767px`, take a look 👇🏽 

![Screenshot 2023-02-24 at 21 50 18](https://user-images.githubusercontent.com/45739963/221332435-85024db1-0b51-4102-8f0c-59fc2d503d96.png)
--------
![Screenshot 2023-02-24 at 21 51 49](https://user-images.githubusercontent.com/45739963/221332504-aa08349d-bdd9-45ff-bb02-72baaf064b5d.png)